public/pages/DetectorResults/containers/AnomalyResultsLiveChart.tsx

```diff
@@ -68,6 +71,7 @@ export const AnomalyResultsLiveChart = (
 ) => {
   const dispatch = useDispatch();
 
+  const [firstLoading, setFirstLoading] = useState<boolean>(true);
```

Comment:
small: maybe change to something like `liveChartLoading` or `chartLoading` to be more clear on what is loading here?